docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -761,6 +761,18 @@ and
 - `SEND_BUFFER_LEN`: **100**: Buffer length of mailing queue. **DEPRECATED** use `LENGTH` in `[queue.mailer]`
 - `SEND_AS_PLAIN_TEXT`: **false**: Send mails only in plain text, without HTML alternative.
 
+## Override Email Headers (`mailer.override_header`)
+
+⚠️ This is empty by default, use it only if you know what you need it for.
```

Comment:
https://docs.gitea.com/1.21/administration/config-cheat-sheet?_highlight=⚠️#mailer-mailer

@silverwind 

we have no "note" func in the docs ... but the utf8 emoji is already in use ...

Reply:
Ok I guess then it's fine, but I would be surprised if docusaurus can't do this or has no plugin for it 😆.

Reply:
https://github.com/go-gitea/gitea/pull/27860/commits/776c550ff22987b46c4ac4123cc608bf855380d2

but I can not verify it works withour setup ...jet